### DISCUSSION TITLE
Raster offset

### DISCUSSION THREAD

**Discussion by jheslin** [corteva/geocube#146] [Q&A]
I have a geodataframe of points that I am converting to a raster. Using the make_geocube function, I am able to create a raster but noticed that there is a consistent offset between the points and the raster cells. I would expect the points to represent the centroid of each corresponding raster cell.

Solutions I have tried:
- Converted the points to a polygon geodataframe via [this suggestion on StackExchange](https://gis.stackexchange.com/questions/403964/point-file-to-raster-conversion-using-geocube), but I still get the same the same offset.
- Used the align parameter in the make_geocube function. I figured out the values that would fix the offset would have to be [-5.5, 5], but I am given an error when trying to use a negative number in the array.

Appreciate any advice on this issue!

**Comment by snowman2** [comment]
Do you have a re-producible code snippet? Are you able to provide input files?
